### PR TITLE
[TRANSLATION] Added Polish DWNL translation

### PR DESCRIPTION
**Pull request by Mcpg**
## Purpose

_Translated network dwnl utility to Polish._

### CODE REVIEW

File: base/applications/network/dwnl/lang/pl-PL.rc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    IDS_BYTES_DOWNLOADED "Pobrano %I64u bajt(ów)."
+    IDS_BYTES_DOWNLOADED_FULL "%2d%% (pobrano %I64u bajt(ów))"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_ERROR_DOWNLOAD "\nBŁĄD: Pobieranie nieudane.\n"
+    IDS_ERROR_CODE "\nBŁĄD: %u\n"
+    IDS_ERROR_FILENAME "\nBŁĄD: Nie można ustalić nazwy pliku, proszę o podanie docelowej nazwy pliku.\n"
+    IDS_ERROR_PROTOCOL "\nERROR: Niewspierany protokół.\n"
```
Comment (binarymaster) [inline base/applications/network/dwnl/lang/pl-PL.rc]: Last one `ERROR` is not translated. :wink:

File: base/applications/network/dwnl/lang/pl-PL.rc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    IDS_SEND_REQUEST "Wysyłanie zapytania... "
+    IDS_BYTES_DOWNLOADED "Pobrano %I64u bajt(ów)."
+    IDS_BYTES_DOWNLOADED_FULL "%2d%% (pobrano %I64u bajt(ów))"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_ERROR_DOWNLOAD "\nBŁĄD: Pobieranie nieudane.\n"
+    IDS_ERROR_CODE "\nBŁĄD: %u\n"
+    IDS_ERROR_FILENAME "\nBŁĄD: Nie można ustalić nazwy pliku, proszę o podanie docelowej nazwy pliku.\n"
```
Comment (archeYR) [inline base/applications/network/dwnl/lang/pl-PL.rc]: Well, it's correct, but I would translate it as "...podaj docelową nazwę pliku." to make it consistent with ReactOS/Windows translations.

File: base/applications/network/dwnl/lang/pl-PL.rc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "Użycie: dwnl URL [CEL]\n"
+    IDS_RESOLVING "Rozwiązywanie %s... "
+    IDS_DONE "ukończone.\n"
+    IDS_LENGTH_FULL "Długość: %I64u [%s]\n"
+    IDS_LENGTH "Długość: %I64u\n"
+    IDS_FILE_SAVED "\nPlik zapisany.\n"
+    IDS_CONNECTING_TO_FULL "Łączenie się z %s [%s]... "
```
Comment (archeYR) [inline base/applications/network/dwnl/lang/pl-PL.rc]: I would suggest "Nawiązywanie połączenia z...".

File: base/applications/network/dwnl/lang/pl-PL.rc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    IDS_CONNECTING_TO "Łączenie się z %s... "
```
Comment (archeYR) [inline base/applications/network/dwnl/lang/pl-PL.rc]: Same.

**Pull request merged**